### PR TITLE
Add Python Errors Inbox docs (MERGE 3/31 AM)

### PR DESCRIPTION
**Pull request by lrafeei**
This PR adds documentation for the Python Agent's Errors Inbox APIs, the `set_user_id` API and the `set_error_group_callback` API.

### CODE REVIEW

**Review by zuluecho9 — DISMISSED** [review]

File: src/content/docs/apm/agents/python-agent/python-agent-api/seterrrorgroupcallback-python-agent-api.mdx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+## Example usage [#examples]
+
+### Set error group callback [#error-group-callback-example]
+
+An example of using `set_error_group_callback`:
+
+```
+def customer_callback(exc):
+  if isinstance(exc,ValueError):
+    return "value"
```
Comment (hmstepanek) [inline src/content/docs/apm/agents/python-agent/python-agent-api/seterrrorgroupcallback-python-agent-api.mdx:119]:
```suggestion
    return "group1"
```

File: src/content/docs/apm/agents/python-agent/python-agent-api/seterrrorgroupcallback-python-agent-api.mdx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  except Exception:
+    newrelic.agent.notice_error()
+
+
+def example_function(customer_callback):
+  try:
+    newrelic.agent.set_error_group_callback(customer_callback)
+    some_other_function()
+  finally:
+    set_error_group_callback(None)
```
Comment (hmstepanek) [inline src/content/docs/apm/agents/python-agent/python-agent-api/seterrrorgroupcallback-python-agent-api.mdx:134]:
```suggestion
    newrelic.agent.set_error_group_callback(None)
```

File: src/content/docs/apm/agents/python-agent/python-agent-api/setuserid-python-agent-api.mdx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+* Nothing was provided as an input to the API.
+* Something was provided but it was not a string.
+
+## Example usage [#examples]
+
+### Set user ID inside a background task [#set-user-id-example]
+
+An example of using `set_user_id` in a background task:
+
+```
```
Comment (hmstepanek) [inline src/content/docs/apm/agents/python-agent/python-agent-api/setuserid-python-agent-api.mdx:83]: If it's a background task then we don't need to create an application like this in our example. Maybe just change the wording above to be "An example of using `set_user_id` inside a simple flask app to set the user id of the transaction".

File: src/content/docs/apm/agents/python-agent/python-agent-api/setuserid-python-agent-api.mdx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+  from flask import Flask
+  import newrelic.agent
+
+  app = Flask("Flask Test App")
+
+  @newrelic.agent.background_task()
```
Comment (hmstepanek) [inline src/content/docs/apm/agents/python-agent/python-agent-api/setuserid-python-agent-api.mdx:89]:
```suggestion
```

File: src/content/docs/apm/agents/python-agent/python-agent-api/setuserid-python-agent-api.mdx [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+  @app.route("/")
+  def hello():
+    try:
```
Comment (hmstepanek) [inline src/content/docs/apm/agents/python-agent/python-agent-api/setuserid-python-agent-api.mdx:92]: Change this to the following:
```
def hello():
    newrelic.agent.set_user_id("my-user-id")
    return "Hello World!"
```

**Review by hmstepanek — CHANGES_REQUESTED** [review]
Requested some changes to improve understanding. Overall looks good though!

**Pull request merged**